ddl/reorg.go

```diff
@@ -66,6 +66,10 @@ type reorgCtx struct {
 
 	doneKey atomic.Value // nullable kv.Key
 
+	// element is used to record the current element in the reorg process, it can be
+	// accessed by reorg-worker and daemon-worker concurrently.
+	element atomic.Value
```

Comment:
Why not convert `reorgInfo.currElement` to `atomic.Value`?

Reply:
yes, we could, but it seems not elegant. 
reorgCtx is used to store the process of reorg job, like `doneKey`, once the timeout is triggered, the reorgCtx process should be stored into KV. Put it here seems reasonable.

Reply:
ok